packages/salesforcedx-vscode-lwc/src/commands/lightningLwcPreview.ts

```diff
@@ -144,24 +143,31 @@ export const lwcPreview = async (sourceUri: URI) => {
     return;
   }
 
-  if (!fs.existsSync(resourcePath)) {
-    const message = nls.localize('lightning_lwc_preview_file_nonexist', resourcePath);
-    showError(new Error(message), logName, commandName);
-    return;
-  }
+  try {
+    const stat = await vscode.workspace.fs.stat(sourceUri);
```

Comment:
checking my understanding: switching from resourcePath (path) to sourceUri (uri) because that's what the fs.api stuff needs?

Reply:
yes!